pkg/identityserver/rights.go

```diff
@@ -168,6 +168,9 @@ func (is *IdentityServer) GatewayRights(ctx context.Context, gtwIDs ttnpb.Gatewa
 	if err != nil {
 		return nil, err
 	}
+	// Clear the EUI to ensure that the query only considers the Gateway ID.
```

Comment:
I think I'd prefer to have even before `is.getRights`. Even though `getRights` doesn't really look at it, I think we should avoid any confusion.